haystack/modeling/model/adaptive_model.py

```diff
@@ -630,6 +630,8 @@ def __init__(
         """
         import onnxruntime
 
+        super().__init__(prediction_heads)
```

Comment:
I introduced this `super` call because a subclass should always call `super`, even if it's useless, to be future-proof. There's no issue with the ordering though. I could import `onnxruntime` after the `super` call, I don't think it makes a difference. Do you think it does?

Reply:
No, not really. It might just look like it's important to have it first. Maybe that's only my subjective recognition of the super-init call placed between the other init code because it's highlighted in this PR.